src/http/plugins/tenant-feature.ts

```diff
@@ -0,0 +1,29 @@
+import fastifyPlugin from 'fastify-plugin'
+import { getConfig } from '../../config'
+import { Features, getFeatures } from '../../database/tenant'
+
+/**
+ * Middleware that requires a specific feature to be enabled
+ * for a given tenant.
+ *
+ * This only applies for multi-tenant-application.
+ * for single tenant, use environment variables to toggle features
```

Comment:
nits: 
- `For` (capitalized)
- consistent use of full stops to end sentences
- consistent hyphenation of multi-tenant and single-tenant